core/src/test/java/kafka/log/remote/RemoteLogReaderTest.java

```diff
@@ -40,18 +42,29 @@
 import static org.mockito.Mockito.when;
 
 public class RemoteLogReaderTest {
+    public static final String TOPIC = "test";
     RemoteLogManager mockRLM = mock(RemoteLogManager.class);
+    BrokerTopicStats brokerTopicStats = new BrokerTopicStats();
     LogOffsetMetadata logOffsetMetadata = new LogOffsetMetadata(100);
     Records records = mock(Records.class);
 
+    @BeforeEach
+    public void setUp() {
+        KafkaYammerMetrics.defaultRegistry()
+            .allMetrics()
+            .keySet()
+            .forEach(metricName -> KafkaYammerMetrics.defaultRegistry().removeMetric(metricName));
```

Comment:
You can use `TestUtils.clearYammerMetrics()`

Reply:
Thanks, will change this.